api/v1alpha1/virtualmachine_types.go

```diff
@@ -320,8 +328,31 @@ type VirtualMachineSpec struct {
 	// instance.  See VirtualMachineClass for more description.
 	ClassName string `json:"className"`
 
-	// PowerState describes the desired power state of a VirtualMachine.  Valid power states are "poweredOff" and "poweredOn".
-	PowerState VirtualMachinePowerState `json:"powerState"`
+	// PowerState describes the desired power state of a VirtualMachine.
+	//
+	// Please note this field may be omitted when creating a new VM and will
+	// default to "poweredOn." However, once the field is set to a non-empty
+	// value, it may no longer be set to an empty value.
+	//
+	// Additionally, setting this value to "suspended" is not supported when
+	// creating a new VM. The valid values when creating a new
+	// VM are "poweredOn" and "poweredOff." An empty value is also allowed on
+	// create since this value defaults to "poweredOn" for new VMs.
+	//
+	// +optional
+	PowerState VirtualMachinePowerState `json:"powerState,omitempty"`
+
+	// PowerOffMode describes the desired behavior when powering off a VM.
+	//
+	// By default a VM is powered off with the equivalent of a halt operation,
```

Comment:
I had thought of TryGraceful as well, but the problem with changing the default now is it would greatly impact TKG without them changing something. A graceful shutdown of a Kube node takes a rather long time.

Reply:
@bryanv How about we change the default behavior for v1a2 then? That way as long as TKG is still pointing to v1a1, they're using halt by default?